std/cpp/NativeXml.hx

```diff
@@ -405,13 +404,13 @@ class Xml {
 	}
 
 	static function __init__() : Void untyped {
-		PCData = Type.createEnum(XmlType,"__");
-		Element = Type.createEnum(XmlType,"__");
-		CData =  Type.createEnum(XmlType,"__");
-		Comment = Type.createEnum(XmlType,"__");
-		DocType = Type.createEnum(XmlType,"__");
-		ProcessingInstruction =  Type.createEnum(XmlType,"__");
-		Document = Type.createEnum(XmlType,"__");
+		PCData = Type.createEnum(cast XmlType,"__");
```

Comment:
It probably doesn't, I wanted to ask @hughsando about what I'm supposed to do with this. This empty `XmlType` enum is pretty terrible to handle in general.